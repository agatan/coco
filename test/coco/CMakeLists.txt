add_subdirectory(expected)
add_subdirectory(combix)
