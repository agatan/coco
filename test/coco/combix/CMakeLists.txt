add_executable(stream_test stream_test.cpp)
add_test(NAME stream_test COMMAND $<TARGET_FILE:stream_test>)
target_link_libraries(stream_test
  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
  )

add_executable(primitives_test primitives_test.cpp)
add_test(NAME primitives_test COMMAND $<TARGET_FILE:primitives_test>)
target_link_libraries(primitives_test
  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
  )

add_executable(combinators_test combinators_test.cpp)
add_test(NAME combinators_test COMMAND $<TARGET_FILE:combinators_test>)
target_link_libraries(combinators_test
  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
  )

add_executable(parser_test parser_test.cpp)
add_test(NAME parser_test COMMAND $<TARGET_FILE:parser_test>)
target_link_libraries(parser_test
  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
  )
