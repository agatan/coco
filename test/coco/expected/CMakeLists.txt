add_executable(expected_test expected_test.cpp)
add_test(NAME expected_test COMMAND $<TARGET_FILE:expected_test>)
target_link_libraries(expected_test
  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
  )
